Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.h

```diff
@@ -21,22 +21,21 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface VOKFetchedResultsDataSource : NSObject <NSFetchedResultsControllerDelegate, UITableViewDelegate, UITableViewDataSource>
 
-@property (readonly) Class managedObjectClass;
-@property (weak, readonly) UITableView *tableView;
-@property (weak, readonly) NSManagedObjectContext *managedObjectContext;
+@property (nonatomic, assign, readonly) Class managedObjectClass;
+@property (nonatomic, weak, readonly) UITableView *tableView;
 
-@property (weak) id<VOKFetchedResultsDataSourceDelegate> delegate;
+@property (nonatomic, weak) id<VOKFetchedResultsDataSourceDelegate> delegate;
 
-@property (strong, readonly) NSFetchedResultsController *fetchedResultsController;
+@property (nonatomic, strong, readonly) NSFetchedResultsController *fetchedResultsController;
```

Comment:
Question for anyone reading:
`fetchedResultsController`, `managedObjectClass`, and `tableView` are declared as `readonly`
I didn't think instance variables, setters, and getters would be auto-synthesized. And yet, they are!
In the m file `_fetchedResultsController` exists and appears to work correctly. Same with the other readonly properties.